ita-social-projects/GreenCityUBS
# 5520 change request to 2865286637064977289828992900 29013707542054173492 clarification order validation and order details

dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)
* https://github.com/ita-social-projects/GreenCity/issues/5520

## Code reviewers

- [ ] @ABbondar 
- [ ] @KharchenkoV 
- [ ] @juseti 
- [ ] @MarkAndzh 
- [ ] @sergoliarnik 
- [ ] @JJJazl 
- [ ] @MaksymGolik 
- [ ] @Markiro1 
- [ ] @LiliaMokhnatska 

## Summary of change
Added new notification templates and platforms
Added new method to NotificationService
Changed UBSManagementServiceImpl

##Testing Approach
- Log in as user
- Make an order and don`t pay for it.
- Log in as admin
- Change order status to Brought by Himself
- You should receive email about changing status


## CHECK LIST
